### PR TITLE
Finalizing

### PR DESCRIPTION
**Pull request by jjtapley**

### CODE REVIEW

**Comment by jjtapley** [comment]
I merged 'finalizing' into 'select user image' branch - I did this as my local repo wasn't showing the correct css file, but sorry if I did this wrong

File: addQuote.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+<?php
+if (isset($_POST['quote']) && isset($_POST['charName']) && isset($_POST['epName']) && isset($_POST['epNum']) && isset($_POST['season']) && isset($_POST['rating'])) {
+    $quote = '"' . $_POST['quote'] . '"';
+    $character = $_POST['charName'];
+    $episode = "'" . $_POST['epName'] . "'" . " - Season " . $_POST['season'] . ", Episode " . $_POST['epNum'];
+    $rating = $_POST['rating'];
+    $image = selectImage($character, $characterPics);
+    sendImageData($db, $image);
+    sendQuotesData($db, $quote, $character, $episode, $rating);
+    echo '<h1>Quote Successfully Added!</h1>';
```
Comment (snack-attack) [inline addQuote.php:128]: It's a shame the redirect wasn't working for you! But better to get working code into master then refactor with that functionality if you have time!

File: selectImageFunction.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+require_once '.' . DIRECTORY_SEPARATOR . 'characterPicsURLs.php';
+
+/**Function takes the $characterPics array and chooses as random picture of the character based on the user input $character. It returns the url, which can be sent to DB
+ *
+ * @param string $character
+ * @param array  $characterPics
+ *
+ * @return string
+ */
+function selectImage(string $character, array $characterPics) :string {
```
Comment (snack-attack) [inline selectImageFunction.php:12]: Missing a unit test!

File: sendImageDataFunction.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**Function
```
Comment (snack-attack) [inline sendImageDataFunction.php:3]: There's gotta be a better way to describe this function 😏

File: sendQuotesDataFunction.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+//require_once '.' . DIRECTORY_SEPARATOR . 'addQuote.php';
```
Comment (snack-attack) [inline sendQuotesDataFunction.php:3]: Delete commented code - should never go into master

**Review by snack-attack — CHANGES_REQUESTED** [review]
Looks good Jess! Just a few things to fix 😄

**Pull request merged**